packages/fast-usdc/src/types.ts

```diff
@@ -143,6 +143,8 @@ export type AddressHook = {
   query: {
     /** end user destination address */
     EUD: string;
+    /** chain id for end user destination. necessary if EUD is not bech32 */
```

Comment:
in [CAIP-10]https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md#syntax parlance,:
`CID`: `chain_id` (à la CAIP-2)
`EUD`: `account_address`

It also has an `account_id` definition (`chain_id + ":" + account_address`). I propose the "End User Destination" be that full "account_id". The enclosing object is a "query" and it's not sensical to change CID independent of EUD. IOW, EUD needs to specify the full chain-agnostic account ID.

For backwards compatibility we can make the `chain_id` implied.